src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -77,7 +77,7 @@ public class XmlLoggingConfiguration : LoggingConfigurationParser
         /// Initializes a new instance of the <see cref="XmlLoggingConfiguration" /> class.
         /// </summary>
         /// <param name="fileName">Configuration file to be read.</param>
-        public XmlLoggingConfiguration([CanBeNull] string fileName)
+        public XmlLoggingConfiguration([NotNull] string fileName)
```

Comment:
looks like a change which is not suitable for a path version? (breaking behavior change)

Reply:
Well the `[CanBeNull]` is a bug. You cannot load an XmlLoggingConfiguration from a null-filename.

Reply:
If we're sure we wont break stuff, I'm happy :) 👍